.github/workflows/main.yml

```diff
@@ -0,0 +1,24 @@
+name: CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v1
+
+    - name: Install dependencies
+      run: |
+        npm install --prefix packages/react
```

Comment:
I'd personally recommend aligning and using `yarn` everywhere instead, that will prevent future debugging headaches

In using the root alias, you'd saved multiples lines and do a _one for all_

Reply:
Done ✅